docs/interpreter/flink.md

```diff
@@ -266,6 +266,7 @@ Flink in Zeppelin supports 3 execution modes (`flink.execution.mode`):
 * Local
 * Remote
 * Yarn
+* Yarn Application
```

Comment:
Lines 262 and 264 also seem to need correction.
```
## Execution mode (Local/Remote/Yarn)

Flink in Zeppelin supports 3 execution modes (`flink.execution.mode`):
```

Reply:
Fixed